docs/setup/docker.asciidoc

```diff
@@ -9,14 +9,14 @@ found on https://github.com/elastic/kibana-docker/tree/{branch}[GitHub].
 
 === Image types
 
-The images are available in two different configurations or "flavors". The
-`x-pack` flavor, which is the default, ships with X-Pack features
-pre-installed. The `oss` flavor does not include X-Pack, and contains only
-open source Kibana.
-
-NOTE: https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] is
-pre-installed in the default image. With X-Pack installed, Kibana expects to
-connect to an Elasticsearch cluster that is also running X-Pack.
+By default, the images contain both open source and commercial features. 
```

Comment:
Without more context, this statement kind of gives the impression that you must pay for a license to use the default image, but that's not the case. The term "commercial" is appropriate, but in the vast majority of situations that term refers to things that cost money, and that's not the case here.

Perhaps we can use this opportunity to elaborate about how the images come baked in with all of our free features by default, including apache2 licensed features and elastic licensed features. Folks can also choose to opt-in to a 30 day free trial of our platinum features as well.

Then alternatively folks can get oss-only images.